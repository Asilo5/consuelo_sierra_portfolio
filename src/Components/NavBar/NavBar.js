import React from 'react';
import './NavBar.scss';
import { NavLink } from 'react-router-dom';

const NavBar = () => {
    return (
        <nav>
            <h1>Consuelo</h1>
            <section>
                <NavLink to='/about' className='nav-link' activeClassName='active-nav' >About</NavLink>
                <NavLink to='/projects' className='nav-link' activeClassName='active-nav' >Projects</NavLink>
                <NavLink to='/contact' className='nav-link' activeClassName='active-nav' >Contact</NavLink>
            </section>
        </nav> 
    )
}

export default NavBar;