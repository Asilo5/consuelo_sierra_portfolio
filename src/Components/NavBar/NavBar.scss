nav {
    display: flex;
    justify-content: flex-end;
    padding: 20px;
    background: #4CADA0;
    margin-bottom: -2em;

    .nav {
        margin-left: 5em;
        text-decoration: none;
        color: navajowhite;
        font-size: 18px;
        border: 1px solid transparent;
        padding: 5px;
        transition: 1s ease-in-out;
      
        &:hover {
            border-bottom: 1px solid navajowhite;
        }
      }

    .active {
        border-bottom: 1px solid navajowhite;
      }
}

