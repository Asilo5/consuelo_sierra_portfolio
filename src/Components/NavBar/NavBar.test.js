import React from 'react';
import { shallow } from 'enzyme';
import NavBar from './NavBar';

describe('NavBar', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<NavBar />)
    })

    it('should match snapshot with all information passing in correctly', () => {
        expect(wrapper).toMatchSnapshot();
    })
})