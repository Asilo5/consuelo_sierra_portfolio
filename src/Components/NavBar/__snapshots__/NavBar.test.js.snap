// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar should match snapshot with all information passing in correctly 1`] = `
<nav>
  <h1>
    Consuelo Sierra
  </h1>
  <NavLink
    activeClassName="active-nav"
    to="/about"
  >
    About
  </NavLink>
  <NavLink
    activeClassName="active-nav"
    to="/projects"
  >
    Projects
  </NavLink>
  <NavLink
    activeClassName="active-nav"
    to="/contact"
  >
    Contact
  </NavLink>
</nav>
`;
